src/cms/src/api/management-document/content-types/management-document/schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "kind": "collectionType",
+  "collectionName": "management_documents",
+  "info": {
+    "singularName": "management-document",
+    "pluralName": "management-documents",
+    "displayName": "Management-document",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "attributes": {
+    "title": {
+      "type": "string"
+    },
+    "url": {
+      "type": "string"
+    },
+    "description": {
+      "type": "text"
+    },
+    "documentDate": {
+      "type": "date"
+    },
+    "documentType": {
+      "type": "relation",
+      "relation": "oneToOne",
```

Comment:
This should actually be oneToMany (one document type can be associated with many documents).  The ticket is wrong